### PR TITLE
feat(reviews): Implement a review api endpoint

### PR DESCRIPTION
**Pull request by blakedietz**
#### What

Implements a reviews endpoint for storing user feedback.

I've already deployed the endpoint and function to the dev stage.

#### Why

To allow festival attendees to give feedback for the application ux.

### Todo

Once merged into develop a few things need to be done upon merge to master:
- [x] create a prod dynamodb instance
- [ ] `npm run deploy:prod`
  - this needs to be run because a new function will be generated. You'll actually be prompted by serverless if you try to just deploy the singular function.

### Testing instructions

You can test indexing entities with the following in your browser:
- `api.tmf.zone/dev/v1/reviews`
- `api.tmf.zone/dev/v1/reviews/{reviewId}`

You can test creating entities with the following using cirl:
- `curl -H "Content-Type: application/json" -X POST "https://api.tmf.zone/dev/v1/reviews" -d '{"rating": 5: "So dank", "email": "test@gmail.com"}'`

### CODE REVIEW

File: api-fort/serverless.yml [hunk 1]
```diff
@@ -129,6 +130,24 @@ functions:
           path: v1/playlists/{playlistId}
           integration: lambda
           cors: true
+  reviews:
+    handler: src/api/reviews/index.router
+    events:
+      - http:
+          method: get
```
Comment (elloboblanco) [inline api-fort/serverless.yml]: Do we really want to GET these? Possibility of leaking email addresses that have been submitted?

Comment (blakedietz) [reply api-fort/serverless.yml]: There are two paths forward: remove email, or remove the get endpoint. I'm impartial to either and will do whatever you prefer.

**Comment by blakedietz** [comment]
Sounds good. I'll update based upon whatever you'd like to do.

**Comment by elloboblanco** [comment]
Let’s just remove the GET for now and collect the data this year.

**Comment by blakedietz** [comment]
Updated based upon your feedback.

**Pull request merged**